.github/workflows/tests.yml

```diff
@@ -98,12 +98,12 @@ jobs:
         run: find $PWD -type f \( -name 'WORKSPACE' -o -name '*.bzl' -o -name '*.bazel' \) | xargs bazel run buildifier -- -lint=fix && git diff --exit-code
       - name: Check docs
         run: bazelisk run docs && git diff --exit-code docs
-  xcodeproj_tests_xcode_13_4_1:
-    name: .xcodeproj Tests on Xcode 13.4.1
+  xcodeproj_tests_xcode_14_2_0:
```

Comment:
Minor this test and ideally just be called `xcodeproj_tests` - same with the lower one to avoid having to do a rename on each one.